assets/styles/_highlight.scss

```diff
@@ -1,4 +1,4 @@
-/* Background */ .chroma { font-size: .9em; padding-left:10px !important; color: #f8f8f2; background-color: $code-background !important; line-height:1.65em;}
+/* Background */ .chroma { font-size: .9em; padding-left:10px !important; color: #f8f8f2; box-shadow: 5px 5px 10px lightgrey; border-radius: 5px; background-color: $code-background !important; line-height:1.65em;}
```

Comment:
is this intentional?

Reply:
I'll share my opinion once I see the look.